participants/fito.json

```diff
@@ -0,0 +1,33 @@
+{
+    "realName": {
+        "givenName": "Adolfo",
+        "familyName": "Guevara",
+        "placeFamilyNameFirst": false,
+        "hideFamilyNameOnWebsite": false
+    },
+    "githubAccountName": "FitoGT",
+    "company": "Freelancer",
+    "when": {
+        "friday": true,
+        "saturday": false
+    },
+    "iCanTakeNotesDuringSessions": true,
+    "tags": [
+        "React",
+        "TypeScript",
+        "ReactQuery",
+        "TailwindCSS"
+    ],
+    "vegan": false,
+    "vegetarian": false,
+    "allergies": [
+        "Seafood"
+    ],
+    "whatIsMyConnectionToJavascript": "Been developing with JS for a long while :)",
+    "whatCanIContribute": "",
```

Comment:
Please fill out something here, so people can connect with you. Without this, the tests will keep failing and we can't merge you, even if a spot might become free for you on Friday...!